packages/protocol/contracts/governance/Validators.sol

```diff
@@ -1152,23 +1152,14 @@ contract Validators is
     return true;
   }
 
-  function getCanForceDeaffiliation() internal pure returns (bytes32[] memory) {
-    bytes32[] memory res = new bytes32[](4);
-    res[0] = DOWNTIME_SLASHER_REGISTRY_ID;
-    res[1] = DOUBLE_SIGNING_SLASHER_REGISTRY_ID;
-    res[2] = GOVERNANCE_SLASHER_REGISTRY_ID;
-    res[3] = GOVERNANCE_REGISTRY_ID;
-    return res;
-  }
-
   /**
    * @notice Removes a validator from the group for which it is a member.
    * @param validatorAccount The validator to deaffiliate from their affiliated validator group.
    */
   function forceDeaffiliateIfValidator(address validatorAccount)
     external
     nonReentrant
-    onlyRegisteredContracts(getCanForceDeaffiliation())
+    onlyRegisteredContracts(getLockedGold().getSlashingWhitelist())
```

Comment:
we can have an `onlySlasher` that calls a `isSlasher(address)`  on locked Gold.

`isSlasher would internally call `isOneOf`, but from LockedGold, instead of here. That would more efficient in terms of gas